mass_editing/migrations/9.0.1.0.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,14 @@
+---Fields in module 'mass_editing'---
+mass_editing / mass.object              / model_ids (many2many)         : DEL relation: ir.model
+mass_editing / mass.object              / model_list (char)             : NEW
+# NOTHING TO DO
```

Comment:
Hi, 
I'm not expert with that modules, but it seems that model_list and model_ids have the same meaning. I guess that postmigration script should fill model_list.
Don't you think ?